provisioning/transport/amqp/src/AmqpClientConnection.cs

```diff
@@ -42,6 +43,10 @@ internal AmqpClientConnection(Uri uri, AmqpSettings amqpSettings)
 
         private bool _isConnectionClosed;
 
+        private TaskCompletionSource<TransportBase> tcs;
```

Comment:
Nit: please use the naming documented in the coding standards. Privates must be prefixed by "_".